hadoop-ozone/recon/src/main/java/org/apache/hadoop/ozone/recon/ReconServerConfigKeys.java

```diff
@@ -162,6 +162,21 @@ public final class  ReconServerConfigKeys {
   public static final long
       OZONE_RECON_CONTAINER_KEY_FLUSH_TO_DB_MAX_THRESHOLD_DEFAULT = 150 * 1000L;
 
+  public static final String
+      OZONE_RECON_TASK_REPROCESS_MAX_ITERATORS = "ozone.recon.task.reprocess.max.iterators";
+
+  public static final int OZONE_RECON_TASK_REPROCESS_MAX_ITERATORS_DEFAULT = 5;
+
+  public static final String
+      OZONE_RECON_TASK_REPROCESS_MAX_WORKERS = "ozone.recon.task.reprocess.max.workers";
+
+  public static final int OZONE_RECON_TASK_REPROCESS_MAX_WORKERS_DEFAULT = 20;
+
+  public static final String
+      OZONE_RECON_TASK_REPROCESS_MAX_KEYS_IN_MEMORY = "ozone.recon.task.reprocess.max.keys.in.memory";
+
+  public static final int OZONE_RECON_TASK_REPROCESS_MAX_KEYS_IN_MEMORY_DEFAULT = 2000;
+
```

Comment:
Please add some comments and explain : when/how to tune these ?

Also please provide some guidance in javadoc here for memory consumption with an example something like below:

 Resource calculation:
  - FSO + OBS tasks running concurrently = 10 iterators + 40 workers = 50 threads
  - Plus caller thread pools = potentially 100+ threads during reprocess
  - Memory: 2000 keys × ~500 bytes/key × 2 tasks = ~2MB (reasonable)

Can also be added in ozone-default.xml documentation with tuning guidelines.